### ISSUE TITLE
Add remark-lint checker

### ISSUE THREAD

**Issue by PhilippBaschke** [flycheck/flycheck#1237]
[remark-lint](https://github.com/wooorm/remark-lint) is a linter for Markdown. 

I am happy to do the work and create a PR. I just wanted to make sure new checkers are accepted at the moment 😌

**Comment by fmdkdd** [comment]
I think we have started accepting them again, as merging #1072 will take some time.

@cpitclaudel?

**Comment by cpitclaudel** [comment]
Yes please! (https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/) :)

**Comment by cpitclaudel** [comment]
Hi @PhilippBaschke; did you ever get around creating that checker?
